apps/datafeeder/src/styles.css

```diff
@@ -50,6 +50,28 @@ ui-button button[type=button].secondary:hover {
   color: white;
 }
 
+.focus\:border-primary:focus {
+  border-color: var(--color-primary-darker) !important;
+}
+
+my-date-picker .mydp {
+  border-width: 2px;
+  border-color: var(--color-primary);
+}
+
+my-date-picker .mydp:focus-within {
+  border-color: var(--color-primary-darker) !important;
+}
+
+my-date-picker::ng-deep input::ng-deep:focus {
+  border-color: var(--color-primary-darker) !important;
+}
+
+tag-input:focus-within, my-date-picker:focus-within {
+  border-color: var(--color-primary-darker) !important;
+}
```

Comment:
I would try to avoid `!important` if possible. Moreover, try to avoid duplicating declaration  when shared accross selector:
```css
foo,
foo:focus,
foo:focus-within {
  border-color: …;
}
```